packages/source-iottwinmaker/src/scene-module/SceneMetadataModule.spec.ts

```diff
@@ -0,0 +1,132 @@
+import { IoTTwinMakerClient } from '@aws-sdk/client-iottwinmaker';
+import { SceneMetadataModule } from './SceneMetadataModule';
+import { SecretsManagerClient } from '@aws-sdk/client-secrets-manager';
+
+/*eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
blocking: no disabling this rule

Reply:
Removed it. Was copied over from another test file. 
Also, updated the tests to use the mock SDK with proper validations.